pkg/common/path.go

```diff
@@ -13,32 +13,43 @@ import (
 	"sigs.k8s.io/kustomize/kyaml/kio/kioutil"
 )
 
+const stdinDash = "-"
+
 func processPaths(paths []string) genericclioptions.FileNameFlags {
 	// No arguments means we are reading from StdIn
 	fileNameFlags := genericclioptions.FileNameFlags{}
 	if len(paths) == 0 {
-		fileNames := []string{"-"}
+		fileNames := []string{stdinDash}
 		fileNameFlags.Filenames = &fileNames
 		return fileNameFlags
 	}
 
+	// Must be a single directory here; set recursive flag.
 	t := true
 	fileNameFlags.Filenames = &paths
 	fileNameFlags.Recursive = &t
 	return fileNameFlags
 }
 
+// DemandOneDirectoryOrStdin processes "paths" to ensure the
```

Comment:
This doesn't match the function name `DemandOneDirectory`. Did you intend to rename the function to match the comment?